charts/gardener-extension-os-coreos/values.yaml

```diff
@@ -5,6 +5,15 @@ image:
 
 resources: {}
 
+vpa:
+  enabled: true
+# resourcePolicy:
+#   minAllowed:
+#     cpu: 100m
+#     memory: 128Mi
```

Comment:
Please, explicitly enable some minAllowed values or disable by default the VPA.

Reply:
I "enabled" the vpa because it was enabled before (without minAllowed and in updateMode Auto). There was no possibility change this settings before.

Reply:
Good catch, but still some minAllowed default values makes sense.
I'm recommending:
```
cpu: 50m
memory: 64Mi
```